src/agor/tools/dev_tooling.py

```
@@ -647,39 +267,91 @@ def list_memory_branches(self) -> list[str]:
-        """
-        timestamp = datetime.utcnow().strftime("%Y-%m-%d_%H%M")
-        snapshot_file = (
-            f".agor/snapshots/{timestamp}_{title.lower().replace(' ', '-')}_snapshot.md"
-        )
 
-        # Get current git info
-        success, current_branch = self._run_git_command(["branch", "--show-current"])
-        if not success:
-            current_branch = "unknown"
+# Convenience functions
+def quick_commit_push(message: str, emoji: str = "🔧") -> bool:
```

Ruff (0.11.9)

Redefinition of unused `quick_commit_push` from line 32

(F811)